From b288f99d785c6e0113c5b3d275073653d8c9b9b5 Mon Sep 17 00:00:00 2001
From: Jun Yan <jerrysteve1101@gmail.com>
Date: Tue, 24 Feb 2026 14:38:35 +0800
Subject: [PATCH 11/11] arm64: dts: allwinner: h6: Enable hdmi sound card on
 TaqiCat A01

TaiqiCat A01 board that has HDMI connector can also transmit audio
through it.
Enable HDMI sound card on TaiqiCat A01.

Signed-off-by: Jun Yan <jerrysteve1101@gmail.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-taiqicat-a01.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-taiqicat-a01.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-taiqicat-a01.dts
index df97a8c0eb5f..6a7598940f1e 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-taiqicat-a01.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-taiqicat-a01.dts
@@ -136,6 +136,10 @@ hdmi_out_con: endpoint {
 	};
 };
 
+&i2s1 {
+	status = "okay";
+};
+
 &i2c3 {
 	status = "okay";
 };
@@ -353,6 +357,10 @@ &rtc {
 	clocks = <&ext_osc32k>;
 };
 
+&sound_hdmi {
+	status = "okay";
+};
+
 &spdif {
 	pinctrl-names = "default";
 	pinctrl-0 = <&spdif_tx_pin>;
-- 
2.56.0

